flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvRowDataSerDeSchemaTest.java

```diff
@@ -150,6 +151,24 @@ public void testSerializeDeserializeCustomizedProperties() throws Exception {
 			deserConfig,
 			";");
 		testField(STRING(), "null", "null", serConfig, deserConfig, ";"); // string because null literal has not been set
+		testField(TIME(3), "12:12:12.232", LocalTime.parse("12:12:12.232") , deserConfig , ";");
+		testField(TIME(3), "12:12:12.232342", LocalTime.parse("12:12:12.232") , deserConfig , ";");
+		testField(TIME(3), "12:12:12.23", LocalTime.parse("12:12:12.23") , deserConfig , ";");
+		testField(TIME(2), "12:12:12.23", LocalTime.parse("12:12:12.23") , deserConfig , ";");
+		testField(TIME(2), "12:12:12.232312", LocalTime.parse("12:12:12.23") , deserConfig , ";");
+		testField(TIME(2), "12:12:12.2", LocalTime.parse("12:12:12.2") , deserConfig , ";");
+		testField(TIME(1), "12:12:12.2", LocalTime.parse("12:12:12.2") , deserConfig , ";");
+		testField(TIME(1), "12:12:12.2235", LocalTime.parse("12:12:12.2") , deserConfig , ";");
+		testField(TIME(1), "12:12:12", LocalTime.parse("12:12:12") , deserConfig , ";");
+		testField(TIME(0), "12:12:12", LocalTime.parse("12:12:12") , deserConfig , ";");
+		testField(TIME(0), "12:12:12.45", LocalTime.parse("12:12:12") , deserConfig , ";");
```

Comment:
All of these only test the deserialization. We should also add tests for serialization. I think we can use `testNullableField` for this purpose. 

Besides, we can rename method `testField(fieldType, csvValue, value, deserializationConfig, fieldDelimiter)` to `testFieldSerialization` to be more specific.

Reply:
rename method  `testField(fieldType, csvValue, value, deserializationConfig, fieldDelimiter)` to `testFieldSerialization` ?or `testFieldDeserialization` ?

Reply:
The reason for not using `testNullableField ` can be seen in our previous dialogue.

Reply:
Rename to `testFieldDeserialization`.

Reply:
We only use `testNullableField` to test conversion without precision loss.

Reply:
done